### PR TITLE
697 preserve choice checked value

### LINKED ISSUE #697: Choices exercise should preserve last submission selection
Currently, if you submit a choice exercise, and then you refresh the page, previous selection is not shown.

### PR DESCRIPTION
**Pull request by aguspina**
Fixes #697

### CODE REVIEW

**Comment by aguspina** [comment]
@flbulgarelli please review

File: app/helpers/concerns/with_choices.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module WithChoices
+  def checked(choice, content)
+    content.split(':').include? choice.index.to_s
```
Comment (flbulgarelli) [inline app/helpers/concerns/with_choices.rb]: I would rename it to `checked?`

File: app/helpers/concerns/with_choices.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module WithChoices
+  def checked(choice, content)
```
Comment (flbulgarelli) [inline app/helpers/concerns/with_choices.rb]: Could we add a test for this?

**Comment by aguspina** [comment]
@flbulgarelli please review again!

**Pull request merged**